### ISSUE TITLE
Simple Statistics

### ISSUE THREAD

**Issue by meiamsome** [CodingTrain/Asteroids#43]
It may be nice to implement some statistics such as:

- Lasers Shot
- Accuracy
- Time Spent Boosting / Fuel used

etc.

**Comment by kwichmann** [comment]
Is there anything specifically we need those for? Other than the fun of it of course :-)

At any rate, maybe flag this with a "beginner friendly" tag or something similar. It seems like a nice way to get started contributing something :-)

**Comment by meiamsome** [comment]
Yeah, I was posting it so we have some fairly simple additions people can make. Added a beginner friendly tag & applied!

**Comment by LeastCreative** [comment]
I am not sure if the guy that was training a bot to machine learn how to play this will be using the old version or not. But these statistics could be used like a "fitness" heuristic for the evolutions so we don't end up with something spinning UK circles shooting as fast as possible lol

**Comment by shiffman** [comment]
@LeastCreative Yes, the plan is for @llSourcell to train a bot to play the version that is in this repo!
